x/gov/spec/01_concepts.md

```diff
@@ -140,12 +140,16 @@ casted on a proposal for the result to be valid.
 Threshold is defined as the minimum proportion of `Yes` votes (excluding
 `Abstain` votes) for the proposal to be accepted.
 
-Initially, the threshold is set at 50% with a possibility to veto if more than
-1/3rd of votes (excluding `Abstain` votes) are `NoWithVeto` votes. This means
-that proposals are accepted if the proportion of `Yes` votes (excluding
-`Abstain` votes) at the end of the voting period is superior to 50% and if the
-proportion of `NoWithVeto` votes is inferior to 1/3 (excluding `Abstain`
-votes).
+Initially, the threshold is set at 50% of `Yes` votes, excluding `Abstain`
+votes. A possibility to veto exists if more than 1/3rd of all votes are
+`NoWithVeto` votes.  This means that proposals are accepted if:
+* there exist bonded tokens.
+* quorum has been achieved.
+* the proportion of `Abstain` votes is inferior to 1/1.
+* the proportion of `NoWithVeto` votes is inferior to 1/3, including
+  `Abstain` votes.
+* the proportion of `Yes` votes, excluding `Abstain` votes, at the end of
+  the voting period is superior to 1/2.
```

Comment:
nit: if you're going to punctuate, then you should also capitalize :)

Reply:
Heh, true.  Done and done.